src/pages/index.tsx

```diff
@@ -225,27 +248,6 @@ export const getStaticProps = (async ({ locale }) => {
   }
 }) satisfies GetStaticProps<Props>
 
-const CodeblockSkeleton = () => (
```

Comment:
I cleaned up this code a bit. Moved these 2 components to the top and renamed the `CodeblockSkeleton` to just `Skeleton` (to be reused in the Stats section).